stdlib/public/core/Optional.swift

```diff
@@ -605,6 +731,9 @@ extension Optional {
     }
   }
 }
+#else
+#error("FIXME(NCG): Fill this out.")
```

Comment:
I don't know what configurations don't have borrowing switch, but it may come to bite us.

Reply:
Ah, I guess we'll need to turn this on unconditionally! 😮